### ISSUE TITLE
Fehler: MappingManager file not found in Citavi aber nur mit QuoTB

### ISSUE THREAD

**Issue by thomask1** [Jay-Squared/QuotationsToolbox#12]
Hi.
Hier nochmal der Fehler, welcher bei jedem Titelwechsel (mit PDF als Attachment) und dem AddOn in Citavi erscheint.

Technical Information for the Citavi Team:
pdftron.Common.PDFNetException: Exception: 
	Message: MappingManager file not found: PFAD\TITEL.pdf
	Conditional expression: fs::exists(some_path)
	Version    : 7.0.1.72235
	Filename   : MappingManager.cpp
	Function   : trn::Filters::MappingManager::Init
	Linenumber : 116
bei pdftron.PDF.PDFDoc..ctor(String filepath)
bei SwissAcademic.Pdf.Document..ctor(String path)
bei QuotationsToolbox.PDFUtilities.GetPageWidths(IEnumerable`1 locations)
bei QuotationsToolbox.QuotationsSorter.SortActiveReferenceQuotations()
bei QuotationsToolbox.UI.MainForm_ActiveReferenceChanged(Object o, EventArgs a)
bei SwissAcademic.Citavi.Shell.MainForm.OnActiveReferenceChanged()
bei SwissAcademic.Citavi.Shell.MainForm.ChangeActiveReference(Reference reference)
bei SwissAcademic.Citavi.Shell.MainForm.referenceEditorNavigationGrid_AfterRowActivate(Object sender, EventArgs e)
bei Infragistics.Win.UltraWinGrid.UltraGrid.OnAfterRowActivate()
bei Infragistics.Win.UltraWinGrid.UltraGrid.FireEvent(GridEventIds id, EventArgs e)
bei Infragistics.Win.UltraWinGrid.UltraGrid.OnActiveRowChange(UltraGridRow newActiveRow, Boolean scrollIntoView)
bei Infragistics.Win.UltraWinGrid.UltraGridBase.SetActiveRow(UltraGridRow row, Boolean scrollIntoView)
bei Infragistics.Win.UltraWinGrid.UltraGridRow.Activate()
bei Infragistics.Win.UltraWinGrid.UltraGridRow.SetFocusAndActivate(Boolean byMouse, Boolean enterEditMode, Boolean byTabKey)
bei Infragistics.Win.SelectionStrategyExtended.OnMouseDown(ISelectableItem item, MouseMessageInfo& msginfo, Boolean forceDrag)
bei Infragistics.Win.ControlUIElementBase.ProcessMouseDownHelper(Object sender, MouseEventArgs e)
bei Infragistics.Win.ControlUIElementBase.ProcessMouseDown(Object sender, MouseEventArgs e)
bei Infragistics.Win.Utilities.ProcessEvent(Control control, ProcessEvent eventToProcess, EventArgs e)
bei Infragistics.Win.UltraWinGrid.UltraGrid.OnMouseDown(MouseEventArgs e)
bei SwissAcademic.Controls.UltraGrid.OnMouseDown(MouseEventArgs e)
bei System.Windows.Forms.Control.WmMouseDown(Message& m, MouseButtons button, Int32 clicks)
bei System.Windows.Forms.Control.WndProc(Message& m)
bei System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

PS: Sorry, das **insert code** funktioniert richtig schlecht...

**Comment by Jay-Squared** [comment]
Ich kann das nicht reproduzieren. Ist "PFAD\TITEL.pdf " das, was da wirklich steht, oder ist das hier absichtlich anonymisiert? Ist die Datei denn an dem angegebenen Pfad vorhanden? Ich kann den Code noch mal durchgehen, und gucken, ob das AddOn irgendwo noch toleranter ob fehlender Dateien werden kann, aber selbst wenn ich eine Datei absichtlich verstecke, passiert das bei mir nicht.

**Comment by thomask1** [comment]
Nein, ist anonymisiert, dort steht der Dateiname des Attachements.
Ja, die Datei ist genau dort vorhanden. Es passiert auch, wenn ich mit deinem AddOn in einer "neuen Citavi-Session" nichts geändert habe, sondern nur einfach zum nächsten Eintrag springe (z. b. von Goethe zur Hölderlin).
Das wundert mich ja auch. Ohne dein AddOn (also Endung umbenannt) gibt Citavi, bei immer noch gleicher Verzeichnissituation, keine Fehler von sich.

**Comment by thomask1** [comment]
PS: Falls es helfen könnte und du hast Zeit, Lust und den ganzen Rest ;-) wäre ein Onlinemeeting möglich. Nur falls..
